openedx/core/djangoapps/bookmarks/apps.py

```diff
@@ -0,0 +1,16 @@
+"""
+Configuration for bookmarks Django app
+"""
+from django.apps import AppConfig
+
+
+class BookmarksConfig(AppConfig):
+    """
+    Configuration class for bookmarks Django app
+    """
+    name = 'openedx.core.djangoapps.bookmarks'
+    verbose_name = "Bookmarks"
+
+    def ready(self):
+        # Register the signals handled by bookmarks.
+        import openedx.core.djangoapps.bookmarks.signals
```

Comment:
Any reason for not simplifying this to `import .signals`?

Reply:
I'm fine with doing that - but I'm a bit confused. Have we decided to import absolute paths at the top of the file? (I seem to remember this.) If so, would that apply here as well?

Personally, I prefer the relative dotted paths as you suggest since it directly indicates that the imported code is local to this module.

Reply:
It probably needs to look like
```python
from . import signals  # pylint: disable=unused-variable
```
which is what we do in other apps.

I don't recall reading anything definitive regarding relative vs absolute imports within edx code.

Reply:
After our chat, I will change this import.